social/src/main/java/org/exoplatform/platform/qa/ui/social/pageobject/MyNotificationsSetting.java

```diff
@@ -999,34 +1000,34 @@ public void checkIntranetNotifiToggle() {
    */
   public void veriftyAllEmailNotiDefaultDisable() {
     info("Verify that daily email is disabled");
-    evt.waitForElementNotPresent(ELEMENT_NEW_USER_SELECTED_BOX_MAIL_ICON.replace("$option",
-            selectBoxMailType.Daily.name().toString()));
+    $(byXpath(ELEMENT_NEW_USER_SELECTED_BOX_MAIL_ICON.replace("$option", selectBoxMailType.Daily.name().toString()))).click();
```

Comment:
Wrong action : evt.waitForElementNotPresent ==> waitUntil(not(visible)...
Same for actions bellow